android/quest/src/test/java/org/smartregister/fhircore/quest/QuestConfigServiceTest.kt

```diff
@@ -57,7 +57,7 @@ class QuestConfigServiceTest : RobolectricTest() {
 
   @Test
   fun testResourceSyncParam_shouldHaveResourceTypes() {
-    every { sharedPreferencesHelper.read(any(), any()) } returns
+    every { sharedPreferencesHelper.read(any(), null) } returns
```

Comment:
Why are we switching to null here?

Reply:
prior sharedpreference isn't handling read/write with boolean, I have to sync sharedPreference with boolean values, to check whether used has stored pin or not, (note pin value further stored with securedSharedPref)

I have also wrote test for this sharedPref class, and updated all modules tests classes using this sharedPref https://github.com/opensrp/fhircore/pull/1030/files#diff-2b482d14ce1d51570cfdc7c9d88900aa46b9bb8f3282d16cddb53e3d23f44934

btw, any() -> also can be considered as passing null value too, which I force pass as the mentioned unit test case is expecting null there,